pdc/apps/compose/templates/compose_detail.html

```diff
@@ -89,6 +89,7 @@ <h3 class="sub-header">{% trans 'Variants' %}</h3>
 .compose-links.dl-horizontal dt {
     width: 4em;
     overflow: visible;
+    text-align: right;
```

Comment:
The commit message is `When the value of Arch with a long string, align right.`. Does here mean it's always align right?